apps/backend/src/modules/users/user.service.ts

```diff
@@ -45,21 +45,25 @@ class UserService implements Service {
 		return item.toObject();
 	}
 
-	public async findAll(): Promise<CollectionResult<UserGetAllItemResponseDto>> {
+	public async findAll(): Promise<CollectionResult<UserItemResponseDto>> {
```

Comment:
Let' rename it to the convention we have; sometimes, we will reuse those types for other methods too, like in this case (we have base type, get by id; get all for now matches, so we use the same time, but maybe we will create another method in the future). By our style guides we need to name it including the action 
```suggestion
	public async findAll(): Promise<CollectionResult<UserGetByIdItemResponseDto>> {
```

Reply:
it's already have been renamed